setup.cfg

```diff
@@ -1,5 +1,5 @@
 [pycodestyle]
-ignore=E501,E741
```

Comment:
W504 and W503 are mutually exclusive and both enabled meaning it's impossible to write a multiline if statement. We follow W503 in other repos so let's do that here.